central/notifiers/cscc/cscc.go

```diff
@@ -232,7 +234,72 @@ func (c *cscc) AlertNotify(ctx context.Context, alert *storage.Alert) error {
 	)
 }
 
-func newCSCC(protoNotifier *storage.Notifier) (*cscc, error) {
+// AlertNotify takes in an alert and generates the notification
+func (c *cscc) InitFinding(ctx context.Context, alert *storage.Alert) (string, *findings.Finding, error) {
```

Comment:
Change to `initFinding` with small I in the beginning. No need to export this func because is used only in this package. The test will be able to access it because it is in the same package.

Reply:
done